.github/workflows/build-git-installers.yml

```diff
@@ -637,9 +637,9 @@ jobs:
         with:
           vault: ${{ secrets.AZURE_VAULT }}
           secrets: |
-            ${{ secrets.LINUX_GPG_KEYGRIP_SECRET_NAME }} > $keygrip
-            ${{ secrets.LINUX_GPG_PRIVATE_SECRET_NAME }} base64> $private-key
-            ${{ secrets.LINUX_GPG_PASSPHRASE_SECRET_NAME }} > $passphrase
+            ${{ secrets.LINUX_GPG_KEYGRIP_SECRET_NAME }}       > $output:keygrip
+            ${{ secrets.LINUX_GPG_PRIVATE_SECRET_NAME }} base64> $output:private-key
+            ${{ secrets.LINUX_GPG_PASSPHRASE_SECRET_NAME }}    > $output:passphrase
```

Comment:
This change probably wanted to be squashed into the previous commit ("fixup! release: add signing step for .deb package" instead of "fixup! build-git-installers: publish gpg public key").